src/middlewared/middlewared/etc_files/local/avahi/avahi-daemon.conf.mako

```diff
@@ -18,10 +18,7 @@
 
     ipv6_enabled = any(middleware.call_sync('interface.ip_in_use', {'ipv4': False, 'ipv6': True}))
 
-    failover_int = middleware.call_sync("failover.internal_interfaces")
```

Comment:
I think we would still want this, @yocalebo can you confirm please ?

Reply:
I also need to bring in some changes from CORE/Enterprise to raise FileShouldNotExist() on passive.